libs/@local/hash-backend-utils/src/machine-actors.ts

```diff
@@ -28,10 +29,29 @@ export type MachineActorIdentifier =
   | GlobalMachineActorIdentifier
   | WebMachineActorIdentifier;
 
+export const getWebMachineId = async (
+  context: { graphApi: GraphApi },
+  authentication: { actorId: ActorEntityUuid },
+  {
+    webId,
+  }: {
+    webId: WebId;
+  },
+): Promise<MachineId> =>
+  context.graphApi.findWeb(authentication.actorId, webId).then(({ data }) => {
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+    if (!data) {
```

Comment:
Similar comment to elsewhere, can we make API types correct or throw an error instead?